### PR TITLE
Minor updates

### LINKED ISSUE #26: Link to GitHub not obvious enough.
There is not enough emphasis placed on the link through to our github repo. This is important to attract new contributors so we should make it more prominent.

### PR DESCRIPTION
**Pull request by mathiascode**
- Added a Github repository link on the main page (fixes #26)
- Added a Forum link to the footer (one user wanted a direct link)
- Removed FAQ link to make room for the Forum link. The FAQ link already exists on the "Support" page, and the FAQ page could be integrated with the User's Manual in the future.
- Makes more sense to link to the GETTING-STARTED.md file instead of the CONTRIBUTING.md file, so I changed it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 <a href="http://builds.cuberite.org/job/Cuberite%20Windows%20x86%20Master/lastSuccessfulBuild/artifact/Install/Cuberite.zip" class="windows">Windows 32-bit</a>
 <a href="http://builds.cuberite.org/job/Cuberite%20Windows%20x64%20Master/lastSuccessfulBuild/artifact/Install/Cuberite.zip" class="windows">Windows 64-bit</a>
 </div>
-<span id="source">or <a href="http://book.cuberite.org/#1.2">compile Cuberite yourself</a></span>
+<span id="source"><a href="https://github.com/cuberite/cuberite">Code Repository</a>&nbsp;&nbsp;&nbsp;•&nbsp;&nbsp;&nbsp;<a href="http://book.cuberite.org/#1.2">Compile Cuberite</a></span>
```
Comment (sphinxc0re) [inline index.html]: I'm not sure about the `nbsp`s. someone?

Comment (mathiascode) [inline index.html]: It does look a bit ugly. I'll create the spacing with CSS instead.

Comment (mathiascode) [inline index.html]: Done.

**Comment by mathiascode** [comment]
Any objections?

**Comment by sphinxc0re** [comment]
My only objection is, that I can't really _review_ the changes :x:

**Comment by mathiascode** [comment]
Here you go: http://final.cuberite.ga/

**Comment by sphinxc0re** [comment]
could you add a nbsp between the 'written', 'in' an the 'C++'?

**Comment by sphinxc0re** [comment]
Also could you use an SVG as the logo

**Comment by mathiascode** [comment]
What exactly do you mean? An extra space?

**Comment by sphinxc0re** [comment]
I mean a 'non-breaking-space' `&nbsp;`

**Comment by mathiascode** [comment]
Reason?

**Comment by sphinxc0re** [comment]
The text breaks badly on huge devices

**Comment by mathiascode** [comment]
Can I see a screenshot? Thanks

**Comment by sphinxc0re** [comment]
Not now because I'm currently on a less huge device :wink:

**Comment by mathiascode** [comment]
Okay. When I have more time, I'll address all text scaling issues in another pull request.

Why a SVG file? The icon is already scalable in the icon font.

**Pull request merged**